be1-go/channel/lao/mod.go

```
@@ -62,19 +66,24 @@ func NewChannel(channelID string, hub channel.HubFunctionalities, msg message.Me
 
 	log = log.With().Str("channel", "lao").Logger()
 
-	inbox := inbox.NewInbox(channelID)
-	inbox.StoreMessage(msg)
+	box := inbox.NewInbox(channelID)
+	box.StoreMessage(msg)
 
-	consensusPath := fmt.Sprintf("%s/consensus", channelID)
+	organizerPk, _ := hub.GetPubkey().MarshalBinary()
```

ignoring errors is dangerous 😬